managers/tests/test_unit_model.py

```diff
@@ -7,14 +7,145 @@
 from freezegun import freeze_time
 from parameterized import parameterized
 
+from managers.commons.constants import ExpectedHoursConstants
 from managers.commons.constants import ProjectConstants
+from managers.commons.strings import ExpectedHoursStrings
+from managers.factories import ExpectedHoursFactory
 from managers.factories import ProjectFactory
+from managers.models import ExpectedHours
 from managers.models import Project
 from users.common.model_helpers import create_user_using_full_clean_and_save
 from users.common.utils import generate_random_string_from_letters_and_digits
+from users.models import CustomUser
 from utils.base_tests import BaseModelTestCase
 
 
+class DataSetUpToTests(BaseModelTestCase):
+    model_class = ExpectedHours
+    required_input = {"date": datetime.now().date(), "user": None, "value": timedelta(hours=8), "project": None}
+
+    def setUp(self):
+        self.user = CustomUser(
+            email="testuser@codepoets.it", password="newuserpasswd", first_name="John", last_name="Doe"
+        )
+        self.user.save()
+
+        self.project = Project(
+            name="Test Project", start_date=datetime.now().date(), stop_date=datetime.now().date() + timedelta(days=30)
+        )
+        self.project.save()
+        self.project.managers.add(self.user)
+        self.project.save()
+
+        self.required_input["user"] = self.user
+        self.required_input["project"] = self.project
+
+        self.SAMPLE_STRING_FOR_TYPE_VALIDATION_TESTS = "String for test"
```

Comment:
You can write this variable using small case letters and just name it:
```python
self.incorrect_value = "incorrect value"
```

Reply:
changed